packages/core/src/components/TextField/TextField.jsx

```diff
@@ -234,17 +233,6 @@ TextField.propTypes = {
    * field's appearance and functionality may be affected.
    */
   mask: PropTypes.oneOf(['currency', 'phone', 'ssn', 'zip']),
-  /**
-   * `max` HTML input attribute
-   */
-  max: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  /**
-   * `min` HTML input attribute
-   */
-  min: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  /**
-   * Whether or not the text field is a multiline text field
-   */
```

Comment:
Did you intend to remove the docstring for the multiline prop as well?

Reply:
oh oops! thanks for the catch